### PR TITLE
adding pypi badges to repository stock lot serial#6825

### PR DESCRIPTION
**Pull request by vishu1993**

### CODE REVIEW

**Comment by PritishC** [comment]
Please fix, changes need to be in a single commit.

File: README.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+Tryton Serialized Inventory Control
+===================================
+
+[![Build Status](https://api.travis-ci.org/openlabs/trytond-stock-lot-serial.svg?branch=develop)](https://travis-ci.org/openlabs/trytond-stock-lot-serial)
+[![Downloads](https://pypip.in/status/openlabs_stock_lot_serial/badge.svg)](https://pypi.python.org/pypi/openlabs_stock_lot_serial/)
+[![Latest Version](https://pypip.in/version/openlabs_stock_lot_serial/badge.svg)](https://pypi.python.org/pypi/openlabs_stock_lot_serial/)
+[![Development Status](https://pypip.in/status/openlabs_stock_lot_serial/badge.svg)](https://pypi.python.org/pypi/openlabs_stock_lot_serial/)
+
+
```
Comment (PritishC) [inline README.md]: Only one blank line.

**Comment by PritishC** [comment]
PR needs to be fixed for merge conflicts.

**Comment by PritishC** [comment]
LGTM.

**Comment by prakashpp** [comment]
@rituparnapanda please merge

**Pull request merged**